src/gui/folderstatusmodel.cpp

```diff
@@ -273,11 +273,15 @@ QVariant FolderStatusModel::data(const QModelIndex &index, int role) const
                 // the error message might contain new lines, the delegate only expect multiple single line values
                 errors.append(legacyError.error().split(QLatin1Char('\n')));
             }
+            if (f->isReady() && f->virtualFilesEnabled() && f->vfs().mode() == Vfs::Mode::WithSuffix) {
+                errors.append({
```

Comment:
Not that I have a better solution, but we may want to make this more visible to users. Maybe we should suggest them to convert to "classic" syncing on startup (i.e., when the accounts and sync folders are loaded, ideally with a single dialog that includes a list of all these directories and a button to directly apply the change.)

Reply:
I guess a user won't want to switch all their 40+ sync folders all at once?

Reply:
Also, who wants to switch 40+ folders manually?

Reply:
I think a huge red box per sync is more than sufficient.